bootcamp-milestone-2/src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import Blog from "@/components/blog";
+import blogs from "@/app/blogData";
+
+export default function BlogPost({ params}: { params: Record<string, string>}) {
+  const slug = params.slug;
+  const blog = blogs.find(blog => blog.slug === `/blog/${slug}`);
+  return blog !== undefined ? (
+    <Blog blog={blog}></Blog>
```

Comment:
Is it worth making a `Blog` component here? You could just have this as the component since you aren't expecting to use the `Blog` component anywhere else.